src/Common/tests/System/Net/Http/LoopbackServer.cs

```diff
@@ -280,6 +281,7 @@ public sealed class Connection : IDisposable
             private Stream _stream;
             private StreamReader _reader;
             private StreamWriter _writer;
+            internal List<string> _requestHeaders;
```

Comment:
I don't think we should add this to the Connection.  We can add helper functions to parse headers, but they should just take the List<string> as a parameter.  And should probably live somewhere else.

Reply:
I can add static function helper to parse lines or I can try to mimic request structure. 
I noticed pattern of processing lines in few places in test code.

Reply:
Yeah, I agree that adding static helpers is worthwhile.  I just don't want to store the request headers on the connection.  Seems unnecessary, and could cause confusion.